src/module_2/download_data_from_s3.py

```diff
@@ -0,0 +1,100 @@
+import boto3
+from dotenv import load_dotenv
+import os
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def load_variables():
```

Comment:
I like this. I would suggest taking all the functions to a separate folder so that you can use them in the future for other modules.